source/assets/sass/components/_callouts.scss

```diff
@@ -19,20 +19,20 @@
   }
 
   &--function {
-    padding-bottom: 0.25rem;
+    padding-bottom: var(--gutter-quarter);
     padding-top: 0;
 
     .signature {
       // Make sure permalinks are visible.
       overflow: visible;
-      margin-left: -1rem;
-      margin-right: -1rem;
+      margin-left: calc(var(--gutter) * -1);
+      margin-right: calc(var(--gutter) * -1);
     }
   }
 
   &--impl-status {
     font-size: 0.8em;
-    margin-top: -1rem;
+    margin-top: calc(var(--gutter) * -1);
     padding: 0.8rem;
```

Comment:
I only made the more common spacing measurements into variables so far so these 0.8rem type of numbers aren't yet in the scale file.